requirements/edx/custom.txt

```diff
@@ -11,15 +11,15 @@
 -e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@82c9dc5e16d10793e8b79e60661e1a78893fce25#egg=xblock-drag-and-drop-v2-new
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@v4.0.7#egg=xblock-ooyala==4.0.7
 git+https://github.com/edx-solutions/xblock-group-project.git@0.1.3#egg=xblock-group-project==0.1.3
--e git+https://github.com/edx-solutions/xblock-adventure.git@0.2#egg=xblock-adventure==0.2
+-e git+https://github.com/edx-solutions/xblock-adventure.git@0.2#egg=xblock-adventure==0.4
```

Comment:
@musmanmalik don't we need to replace xblock-adventure.git@0.2 with xblock-adventure.git@0.4?

Reply:
Thanks for Pointing.